src/adapters/gmx/tests/snapshots/arbitrum.positions.json

```diff
@@ -0,0 +1,129 @@
+{
+  "blockNumber": 150947313,
+  "snapshot": [
+    {
+      "protocolId": "gmx",
+      "name": "GMX",
+      "description": "GMX Liquidity Provider Token adapter",
+      "siteUrl": "https://https://app.gmx.io",
+      "iconUrl": "https://gmx.io//static/media/ic_gmx_40.72a1053e8344ef876100ac72aff70ead.svg",
+      "positionType": "supply",
+      "chainId": 42161,
+      "productId": "glp",
+      "success": true,
+      "tokens": [
+        {
+          "address": "0x4277f8f2c384827b5273592ff7cebd9f2c1ac258",
+          "name": "GMX LP",
+          "symbol": "GLP",
+          "decimals": 18,
+          "type": "protocol",
+          "balanceRaw": "1788213725943696140789934n",
+          "tokens": [
+            {
+              "address": "0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f",
+              "name": "Wrapped BTC",
+              "symbol": "WBTC",
+              "decimals": 8,
+              "type": "underlying",
+              "balanceRaw": "1661250551n",
+              "balance": "16.61250551",
```

Comment:
the numbers seem to be significantly more than what debank is showing 13 vs to 16 btc (your results) https://debank.com/profile/0xbdfa4f4492dd7b7cf211209c4791af8d52bf5c50

Reply:
Thoughts, I guess yours are from 4 days ago

Reply:
<img width="645" alt="image" src="https://github.com/consensys-vertical-apps/mmi-defi-adapters/assets/32621022/874cac48-1328-4a94-9e2b-f850015e34b5">

Reply:
It's due to it being a few days old. Running the command for latest block returns similar results.

![image](https://github.com/consensys-vertical-apps/mmi-defi-adapters/assets/1970725/00067685-0049-4b3c-9860-7624d9c4152f)